README.md

```diff
@@ -64,7 +64,9 @@ USAGE:
 OPTIONS:
     -b, --build ID                   A unique identifier for the build. Should be common among workers participating in the same build.
     -w, --worker ID                  An identifier for the worker. Workers participating in the same build should have distinct IDs.
-    -r, --redis HOST                 Redis host to connect to (default: 127.0.0.1).
+    -r, --redis HOST                 DEPRECATED: Use --redis-host or --redis-url instead. Redis host to connect to (default: 127.0.0.1).
+        --redis-host HOST            Redis host to connect to (default: 127.0.0.1).
+        --redis-url REDIS_URL        Redis URL to connect to (ex: redis://127.0.0.1:6379/0).
```

Comment:
Nit: change `ex` with `e.g.`

Reply:
Also, for consistency, let's use simply `URL` as the name for this variable (because we use `HOST` in `--redis-host`).